.github/workflows/deploy-to-dotorg.yml

```diff
@@ -14,6 +14,44 @@ on:
                     - minor
                     - patch
 jobs:
+    update-google-fonts-json:
+        if: github.repository_owner == 'WordPress'
+        # The type of runner that the job will run on
+        runs-on: ubuntu-latest
+
+        # Steps represent a sequence of tasks that will be executed as part of the job
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: 18
+
+            - name: Get current date
+              id: date
+              run: echo "{date}={$(date +'%Y-%m-%d')}" >> $GITHUB_OUTPUT
```

Comment:
Where is this date used? I couldn't see a reference to it so I was wondering what it was used for.

Reply:
It was just printing the date on the debug log; removing!